### DISCUSSION TITLE
UI/design experiments

### DISCUSSION THREAD

**Discussion by bdarcus** [emacs-citar/citar#201] [Ideas]
You may have noticed that I added another primary face, and a "highlight" one.

As I was experimenting with mockups of what I want to do in my own config, I was wondering about adding one other face, specifically for title, so that we could do this:

![image](https://user-images.githubusercontent.com/1134/127658235-68b7d418-1246-4ee3-ba72-88eb1c7e0e80.png)

Really the only reason to add this, however, is if it helps in scanning the candidate list. I don't think eye-candy alone justifies it.

cc @salutis @mohkale

**Comment by rudy6912** [comment]
Below is how my BibTeX Actions look like with Modus Operandi (where I changed only mode line color). I find the screenshot above much more readable than what I have here:

<img width="1278" alt="Screen Shot 2021-08-01 at 21 35 14" src="https://user-images.githubusercontent.com/1158274/127783215-7fc4cad9-7ffb-44ed-89f0-07d3d7ed2add.png">
